### PR TITLE
feature(dhote): added contributors subsection in README

### LINKED ISSUE #126: Adding Contributors Section in readme.md
Why Contributors section:- A "Contributors" section in a repo gives credit to and acknowledges 
the people who have helped with the project, fosters a sense of community, and helps others
 know who to contact for questions or issues related to the project.

**Issue type**

- [✅] Docs

Expected Outcome :- 

 If you were planning this issue kindly assign it to me! I would love to work on it ! Thank you !

### PR DESCRIPTION
**Pull request by mohitd404**
Fixed #126 
Added Contributors section in Translation in Chinese.

### CODE REVIEW

**Comment by mohitd404** [comment]
@puyuan1996 I have made the changes to to the Readme.zh.md !! Kindly review it !! Thank you !!

**Comment by puyuan1996** [comment]
Hello, for the sake of conciseness, could you please consolidate the modifications of both the Chinese and English versions into one PR? Thank you very much.

**Comment by mohitd404** [comment]
> Hello, for the sake of conciseness, could you please consolidate the modifications of both the Chinese and English versions into one PR? Thank you very much.

As you said I have modification of both the Chinese and English versions into one PR !! I have to closed the Previous PR !!

File: README.zh.md [hunk 1]
```diff
@@ -482,3 +482,9 @@ python3 -u zoo/board_games/tictactoe/config/tictactoe_muzero_bot_mode_config.py
 
 
 <p align="right">(<a href="#top">回到顶部</a>)</p>
+
+### Thank to our contributors 感谢我们的贡献者
```
Comment (puyuan1996) [inline README.zh.md:486]: Hello, what I meant was to use "### Thanks to our contributors" in README.md, and "### 感谢我们的贡献者" in README.zh.md. Haha~. Thank you.

Comment (mohitd404) [reply README.zh.md:486]: Okay Okay I see !! 😀 I'll make it !

Comment (puyuan1996) [reply README.zh.md]: And you can simply add these two changes to the current PR. Thank you.

Comment (mohitd404) [reply README.zh.md]: I did those changes! @puyuan1996  Thank You for your review and guidance !! I appreciate your support !!

Comment (puyuan1996) [reply README.zh.md]: However, it appears that only one file has been modified at the moment.
image

**Comment by mohitd404** [comment]
I'll create a new PR as this one was too messy !! I apologize for that!

**Pull request closed without merging**